### ISSUE TITLE
Put compiled code into "releases"

### ISSUE THREAD

**Issue by talentlessguy** [deadpixi/mtm#31]
Not to waste time on installation people can download already compiled binaries, like this:

```sh
curl -L https://githubusercontent.com/...
```

Please publish releases ❤️ in [this tab](https://github.com/deadpixi/mtm/releases).

**Comment by tenox7** [comment]
Yeah also would be nice to see compiled versions for various versions of unix like AIX, HPUX, Solaris, etc etc.

**Comment by loreb** [comment]
On my computer (cheap pc from 2015 with a rotating hdd) it takes less than two seconds to compile and less than one second to compile with a warm filesystem cache - by far the most expensive operation is installing ncurses if they are missing (assuming the compiler is already there).

TLDR: compilation is _really_ quick, why binaries?

**Comment by tenox7** [comment]
Because some people don't know how to compile, or don't want to bother. Not everyone is a programmer.

**Comment by talentlessguy** [comment]
> On my computer (cheap pc from 2015 with a rotating hdd) it takes less than two seconds to compile and less than one second to compile with a warm filesystem cache - by far the most expensive operation is installing ncurses if they are missing (assuming the compiler is already there).
> 
> TLDR: compilation is _really_ quick, why binaries?

because downloading binaries takes less time than building from source. I think this is evident.

**Comment by loreb** [comment]
@tenox7 I'll admit I didn't even think of that, shame on me! In my defense, I can't imagine someone who uses the terminal, knows what a terminal multiplexer is, is willing to investigate alternatives to screen/tmux AND doesn't have a compiler installed (unless it's common on some of the unix variants you listed, I've only tried some solaris variants in a vm)

@talentlessguy less than two seconds on a cheap laptop with 5400rpm hdd... also, I've **never** tried it myself, but I've seen several questions indicating it may be a PITA, eg https://stackoverflow.com/a/5977518

**Comment by tenox7** [comment]
@loreb Using terminal and a compiler is not comparable. Power users and sysadmins for instance would be very heavily utilizing terminals and even doing some scripting but they would rarely bother compiling anything. This is why users and admins mostly just install packages like deb or rpm instead of compiling from source.

**Comment by talentlessguy** [comment]
@deadpixi any updates on the issue? :thinking:

**Comment by joshbaptiste** [comment]
Indeed.. a static musl release would be nice, trying this out , getting linker errors on FreeBSD just didn't bother and went back to tmux.

**Comment by talentlessguy** [comment]
@joshbaptiste I suggest creating an issue for that, instead of "not bothering"
